src/psyclone/tests/psyir/backend/fortran_test.py

```diff
@@ -552,7 +552,9 @@ def test_fw_kernelschedule(fort_writer, monkeypatch):
 # within previous tests
 
 
-def test_fw_binaryoperator(fort_writer):
+@pytest.mark.parametrize("binary_intrinsic", ["mod", "matmul", "max", "min",
+                                              "sign", "sum"])
+def test_fw_binaryoperator(fort_writer, binary_intrinsic):
     '''Check the FortranWriter class binary_operation method correctly
     prints out the Fortran representation of an intrinsic. Uses sign
```

Comment:
Not any more - you've extended it to do more than that now.

Reply:
Updated text